tests/data_out/data_out_postprocessor_tensor_02.output

```diff
@@ -0,0 +1,40 @@
+    <DataArray type="Float32" Name="strain_xx__strain_xy__strain_yx__strain_yy" NumberOfComponents="9" format="binary">
```

Comment:
Is this the name that shows up in ParaView. This can't be what you want. Also why is this 9 components?

Reply:
Right. That's a separate bug and I already have the fix for that ready and waiting for this PR here to be merged :-)

Reply:
See #10806.